src/components/modals/ManageGroupsAndTypesModal.svelte

```diff
@@ -0,0 +1,686 @@
+              Need to think about the strictness of EE Type and ES Type pairings, and whether the former can exist
+              without an association with the former.-->
+        <CssGridGutter track={1} type="column" />
+        <Panel borderRight padBody={true}>
+          <svelte:fragment slot="header">
+            <SectionTitle>
+              <ExternalSourceIcon slot="icon" />Derivation Groups containing '{selectedExternalEventType.name}'
+            </SectionTitle>
+          </svelte:fragment>
+          <svelte:fragment slot="body">
+            {#if selectedExternalEventTypeDerivationGroups.length > 0}
+              {#each selectedExternalEventTypeDerivationGroups as associatedDerivationGroup}
+                <!-- Collapsible details -->
+                <Collapse
+                  title={associatedDerivationGroup.name}
+                  tooltipContent={associatedDerivationGroup.name}
+                  defaultExpanded={false}
+                >
+                  <svelte:fragment slot="right">
+                    <p class="st-typography-body derived-event-count">
+                      {associatedDerivationGroup.derived_event_total} events
+                    </p>
+                  </svelte:fragment>
+                  <div class="st-typography-body">
+                    <div class="st-typography-bold">Name:</div>
+                    {associatedDerivationGroup.name}
+                  </div>
+
+                  <div class="st-typography-body">
+                    <div class="st-typography-bold">Source Type:</div>
+                    {associatedDerivationGroup.source_type_name}
+                  </div>
+
+                  <Collapse defaultExpanded={false} title="Event Types" tooltipContent="View Contained Event Types">
+                    {#each associatedDerivationGroup.event_types as eventType}
+                      <i class="st-typography-body">{eventType}</i>
+                    {/each}
+                  </Collapse>
+
+                  <Collapse defaultExpanded={false} title="Sources" tooltipContent="View Contained External Sources">
+                    {#each associatedDerivationGroup.sources as source}
+                      <i class="st-typography-body">{source[0]}</i>
+                    {/each}
+                  </Collapse>
+                </Collapse>
+              {/each}
+            {:else}
+              <p class="st-typography-body">No sources containing this event type.</p>
+            {/if}
+          </svelte:fragment>
+        </Panel>
+      {/if}
+    </CssGrid>
+  </ModalContent>
+  <ModalFooter>
+    <div class="filter">
+      <input
+        bind:value={filterString}
+        autocomplete="off"
+        class="st-input w-100"
+        name="filter-ee"
+        placeholder={`Filter ...`}
+      />
+    </div>
+    <button class="st-button secondary" on:click={() => dispatch('close')}> Close </button>
+  </ModalFooter>
+</Modal>
+
+<style>
+  :global(.tab-list.management-tabs-list) {
```

Comment:
Can you add a class containing these global styles to the selector to better encapsulate them? Something like
```
.derivation-groups-modal-content :global(.tab-list.management-tabs-list) { ... }
```